docker-imposm3/Dockerfile

```diff
@@ -18,5 +19,12 @@ RUN pip3 install -r /home/requirements.txt
 
 ADD importer.py /home/
 
+# Add YAML Mapping file
+ADD mapping.yml /home/settings/mapping.yml
```

Comment:
we may need to update this with the one in use for the Maps server: https://github.com/geosolutions-it/osm-styles/blob/master/imposm/mapping.yml

Reply:
the mapping file is changed with the referred above.